src/integration/services.js

```diff
@@ -0,0 +1,340 @@
+module.exports = {
```

Comment:
I think we should use the Cypress config here - I got it working for the `generateRegressionTests` in the `regression-tests-across-all-services-and-pagetypes` branch
https://github.com/bbc/simorgh/pull/6148/files#diff-c9af4078b135bcf96e56bbf5d2d9d152